tests/mappers/test_subworkflow_mapper.py

```diff
@@ -71,23 +70,30 @@ def tearDown(self) -> None:
     @mock.patch("o2a.utils.el_utils.parse_els")
     def test_create_mapper_jinja(self, parse_els):
         # Given
-        parse_els.return_value = self.subworkflow_params
+        parse_els.side_effect = [self.subworkflow_properties, self.configuration_properties]
         # When
         mapper = self._get_subwf_mapper()
 
         # Then
         self.assertEqual("test_id", mapper.task_id)
         self.assertEqual(TriggerRule.DUMMY, mapper.trigger_rule)
         self.assertEqual(self.subworkflow_node, mapper.oozie_node)
-        self.assertEqual(self.main_params, mapper.params)
-        # Propagate config node is present, should forward config properties
-        self.assertEqual({"resourceManager": "localhost:8032"}, mapper.get_config_properties())
+        self.assertEqual(
+            {
+                "examplesRoot": "examples",
+                "nameNode": "hdfs://",
+                "oozie.wf.application.path": "hdfs:///user/pig/examples/pig",
+                "resourceManager": "localhost:8032",
+                "user.name": "potiuk",
```

Comment:
I think, we should avoid store similar information in tests. This can be confused with real data.

Reply:
True. Fixed in f475ebb0a04878d2efa25d171d6a0b0e569af5ff